pylint/checkers/base/name_checker/naming_style.py

```diff
@@ -149,7 +149,8 @@ def _create_naming_options() -> Options:
         help_msg = f"Regular expression matching correct {human_readable_name} names. "
         if name_type in KNOWN_NAME_TYPES_WITH_STYLE:
             help_msg += f"Overrides {name_type_hyphened}-naming-style. "
-        help_msg += f"If left empty, {human_readable_name} names will be checked with the set naming style."
+        help_msg += f"If left empty, {human_readable_name} names will be checked "
+        help_msg += "with the set naming style."
```

Comment:
```suggestion
        help_msg += (
            f"If left empty, {human_readable_name} names will be checked "
           "with the set naming style."
       )
```
To prevent an assignment.